ProcessMaker/Http/Controllers/Process/ScriptController.php

```diff
@@ -48,18 +49,33 @@ public function edit(Script $script, User $users)
         return view('processes.scripts.edit', compact('script', 'selectedUser', 'scriptFormats'));
     }
 
-    public function builder(Script $script)
+    public function builder(Request $request, Script $script)
     {
         $scriptFormat = $script->language_name;
+        $processRequestAttributes = $this->getProcessRequestAttributes();
+        $processRequestAttributes['user_id'] = $request->user()->id;
+        
         $testData = [
-            '_request' => factory(ProcessRequest::class)->raw([
-                'callable_id' => 'process_1',
-                'user_id' => Auth::id(),
-                'process_id' => 1,
-                'process_collaboration_id' => 1,
-            ]),
+            '_request' => $processRequestAttributes
         ];
-
         return view('processes.scripts.builder', compact('script', 'scriptFormat', 'testData'));
     }
+
+    private function getProcessRequestAttributes()
```

Comment:
Does this need to be broken out into its own method? I feel like the problem lay in the factory not the Schema. If we call that factory for something else will the problem come back?